ui/client/datasets/AnomalyDetection.js

```diff
@@ -65,19 +65,19 @@ const RunJob = withStyles(({ spacing }) => ({
 }) => {
```

Comment:
Is this component used anywhere? It doesn't seem to be imported anywhere

Reply:
Not used anymore- this was a PoC for an additional step on the dataset registration process, which would detect anomalies, antipatterns in uploaded datset files (eg too many empty rows, some others) and inform the user.

I don't remember if we explicitly or implicitly completely scratched it out. I recall a decision to temporarily disable. I'm not sure if we're interested in revisiting at some point.

cc @mattprintz , @brandomr